include/core/CMemory.h

```diff
@@ -327,8 +327,8 @@ class CORE_EXPORT CMemory : private CNonInstantiatable {
     }
 
     //! Overload for std::vector.
-    template<typename T>
-    static std::size_t dynamicSize(const std::vector<T>& t) {
+    template<typename T, typename A>
+    static std::size_t dynamicSize(const std::vector<T, A>& t) {
```

Comment:
It's funny that this was missed for `vector` but none of the other containers.

Reply:
I know! I think all the other template arguments for other containers got added after the initial implementation. For some reason it was just missed from vector.